mavlink_vehicles/mavlink_vehicles.hh

```diff
@@ -102,6 +147,20 @@ class mav_vehicle
     global_pos_int home;
     global_pos_int global;
 
+    global_pos_int curr_mission_item_pos;
+    uint16_t curr_mission_item_id = 0;
+    bool curr_mission_item_outdated = true;
+
+    std::vector<global_pos_int> mission_items_list;
+    bool is_sending_mission = false;
+
+    global_pos_int detour_waypoint;
+    bool detour_active = false;
+
+    float rotation_goal = 0;
+    bool rotation_active = false;
+
+    uint8_t system_id = 0;
```

Comment:
Some mavlink messages have a special field that stores the id of the sender: the system_id. Any responses sent by the vehicle that are associated with these messages will have its target_id set as our system_id, what makes possible for us to determine if a reply has been directed to us or to another system. Messages directed to other systems should be ignored.